test/parallel/test-readline-async-iterators.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const join = require('path').join;
+const readline = require('readline');
+const assert = require('assert');
+
+common.crashOnUnhandledRejection();
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const filename = join(tmpdir.path, 'test.txt');
+const file_content = 'abc123\n' +
+                     '南越国是前203年至前111年存在于岭南地区的一个国家\n';
+
+fs.writeFile(filename, file_content);
```

Comment:
Shouldn't this really be `writeFileSync()` to avoid any potential race conditions?

Reply:
Is that a bad practice to use async methods in tests? And this is something that should be fixed?